qa/tasks/vstart_runner.py

```diff
@@ -268,9 +268,19 @@ class LocalRemote(object):
 
     def __init__(self):
         self.name = "local"
-        self.hostname = "localhost"
+        self._hostname = "localhost"
         self.user = getpass.getuser()
 
+    def __del__(self):
+        pass
```

Comment:
Why??

Reply:
Parent class's __del__() closes the SSHClient instance which vstart_runner does not have. Added a comment for this too.